yang/frr-bfdd.yang

```diff
@@ -0,0 +1,387 @@
+module frr-bfdd {
+  yang-version 1.1;
```

Comment:
how do you implement the "no shutdown" facility ?

Reply:
Here:
https://github.com/opensourcerouting/frr/blob/bfdd-northbound/yang/frr-bfdd.yang#L143

And you can also figure it out by looking at the session state:
https://github.com/opensourcerouting/frr/blob/bfdd-northbound/yang/frr-bfdd.yang#L176

You can better visualize this with the command `show yang operational-data /frr-bfdd:bfdd`

```js
{
  "frr-bfdd:bfdd": {
    "bfd": {
      "sessions": {
        "single-hop": [
          {
            "dest-addr": "127.0.0.1",
            "interface": "",
            "vrf": "default",
            "stats": {
              "local-discriminator": 175698406,
              "local-state": "admin-down",
              "local-diagnostic": "ok",
              "local-multiplier": 3,
              "remote-discriminator": 175698406,
              "remote-state": "admin-down",
              "remote-diagnostic": "ok",
              "remote-multiplier": 3,
              "negotiated-transmission-interval": 300000,
              "negotiated-receive-interval": 300000,
              "detection-mode": "async-with-echo",
              "session-down-count": 3,
              "session-up-count": 4,
              "control-packet-input-count": "29708",
              "control-packet-output-count": "29708",
              "negotiated-echo-transmission-interval": 200000,
              "echo-packet-input-count": "44543",
              "echo-packet-output-count": "44546"
            }
          }
        ]
      }
    }
  }
}
```